src/views/open-api-docs-view/types.ts

```diff
@@ -173,6 +181,15 @@ export interface OpenApiDocsPageConfig {
 	 * The product slug is used to fetch product data for the layout.
 	 */
 	productSlug: ProductSlug
+
+	/**
+	 * Optional slug used to theme smaller elements within the service API docs.
+	 * For example, HCP consul will have a `productSlug` of `hcp`,
+	 * but a `serviceProductSlug` of `consul`. If omitted, the `productSlug`
+	 * will be used.
+	 */
+	serviceProductSlug?: ProductSlug
```

Comment:
Optional at the config level, config will fall back to the required `productSlug`.